app/util/math.spec.ts

```diff
@@ -150,17 +134,14 @@ describe('with de-DE locale', () => {
     [-50.2, ['-50', ',2']], // should correctly not round down to -51
     [1000.5, ['1.000', ',5']], // test localeString grouping
   ])('splitDecimal %d -> %s', (input, output) => {
-    expect(splitDecimal(input)).toEqual(output)
+    expect(splitDecimal(input, 'de-DE')).toEqual(output)
   })
 
-  // rounding must work the same irrespective of locale
-  it('round', roundTest)
-
   it.each([
     [0n, ['0', false]],
     [1n, ['1', false]],
     [155n, ['155', false]],
-    [999999n, ['999,999', false]],
+    [999999n, ['999.999', false]],
```

Comment:
This was sort of a bug, but I'm pretty sure it was only due to the way we're mocking this for the tests. I think in a browser it would have worked correctly.